python/pyspark/sql/tests/test_column.py

```diff
@@ -23,6 +23,8 @@
 from pyspark.sql.utils import AnalysisException
 from pyspark.testing.sqlutils import ReusedSQLTestCase
 
+from py4j.protocol import Py4JJavaError
```

Comment:
Can you move `py4j` over `pyspark` per pep8 (https://www.python.org/dev/peps/pep-0008/#imports)?

Reply:
Fixed. Thanks!